src/main/steps/prl-cases/dashboard/utils.ts

```diff
@@ -30,3 +31,36 @@ export const getCasePartyType = (caseData: Partial<CaseWithId>, userId: string):
   }
   return partyType;
 };
+
+export const getCurrentPartyId = (
```

Comment:
change to getPartyId

Reply:
I think try to add partyId prop to getPartyDetails method itself, which will nt hv the need for this method